bdk-ffi/src/bdk.udl

```diff
@@ -441,6 +444,10 @@ interface Wallet {
 
 interface Update {};
 
+interface Policy {
+  string id();
```

Comment:
Is this the only field you need/want for your use case? Just curious really, since we can expose other fields later if needed obv.

Regardless, looks good https://docs.rs/bdk_wallet/latest/bdk_wallet/descriptor/policy/struct.Policy.html

Reply:
Other fields have been exposed now.